### PR TITLE
Analysis related to congresspeople loayal to a single consultancy

### PR DESCRIPTION
**Pull request by cuducos**
Nothing strictly suspicions, but might be some interesting leads for stories.

### CODE REVIEW

**Comment by silviodc** [comment]
Hi @cuducos 

One question about these results.
- Could the deputies to use this subcota to pay for advocacy services?

I didn't find something related to it in the CEAP and we have some results describing it.

**Comment by cuducos** [comment]
> Could the deputies to use this subcota to pay for advocacy services?

2nd article, XI says briefly:

>  contratação, para fins de apoio ao exercício do mandato parlamentar, de consultorias e trabalhos técnicos, permitidas pesquisas socioeconômicas;

AFAIK advocacy are not related to the parliamentary activity _per se_ except if it is to, for example, better understand the consequences of a bill that is being discussed in the House, I guess…

**Comment by silviodc** [comment]
Ohhh!! Yeah i guess you are right...
Its seems a little bit different in the git:

> Contratação para apoio da atividade parlamentar de consultoria e trabalhos técnicos, permitindo pesquisa socioeconômica

File: develop/2017-07-17-cuducos-consultancies.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+# First lets see how many different reimbursements each congressperson had for each consultancy.
+
+# In[4]:
+
+cols = ['applicant_id', 'congressperson_name', 'cnpj_cpf']
+count_per_consultancy = consultancies[cols]             .groupby(cols)             .size()             .to_frame('count_per_consultancy')             .reset_index()             .sort_values('count_per_consultancy', ascending=False)
+count_per_consultancy.head()
+
+
+# ## Couting the total reimbursements congresspeople had in consultancies
```
Comment (jtemporal) [inline develop/2017-07-17-cuducos-consultancies.py]: Typo here =D missing a `n` in `Counting` =P

File: develop/2017-07-17-cuducos-consultancies.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+# This results aren't so helpful, so let's use a minimun of 10 consultancies expenses at the same company, and a ratio of 80% of consultancies expenses done at this same company:
+
+# In[7]:
+
+results = consultancies_grouped     .query('count_per_consultancy >= 10')     .query('percentage >= 0.8')
+results
+
+
+# There are 126 congressperson that are constantly using the same consultancy. This is **not** illegal _per se_ but might be an indicator of something. If anyone wanna go deeper, here are the Jarba links for each of this cases:
```
Comment (jtemporal) [inline develop/2017-07-17-cuducos-consultancies.py]: missing `s` on `Jarbas` ;P

**Comment by cuducos** [comment]
Oops… typos fixed in 9dcec72.

After getting back to this notebook I felt like it's pretty raw — I should spend sometime looking to some of these cases (maybe the top 3, or the top 3 with 100% index as @Irio suggested).

What do you think: merge or label as WIP and we merge after a manual analysis of some of the cases listes in the final table?

**Comment by jtemporal** [comment]
I would totally merge this and open up a new PR based on this one @cuducos

**Comment by jtemporal** [comment]
Let me know what you'd rather do ;)

**Comment by cuducos** [comment]
> I would totally merge this and open up a new PR based on this one @cuducos

If this is useful bi itself, let it be. Merge it ; )

**Pull request merged**